### DISCUSSION TITLE
Add support for RSS feeds

### DISCUSSION THREAD

**Discussion by Unicycle77** [GitbookIO/community#248] [Feature Requests]
### What's your feature/idea?

Requested feature: Add support to enable an RSS feed for specific content (e.g. one or more folders) within a space.

Supporting information: We share product release notes using GitBook, with each release getting its own page within a dedicated release notes folder. Many users have requested the ability to subscribe to these release notes via RSS so that they can see these updates as they are published.

### What's your desired outcome?

If RSS was supported for content updates within specified folders, it would enable users to subscribe to these updates on their own, and it would also enable us to send notifications to users who had opted in to these updates through various channels.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

### Anything else?

The request is to enable it only on individual folders, rather than an entire organization or space.

**Comment by nivm** [comment]
Any other solution/suggestion for customers to subscribe to changes in our gitbook space?

**Reply by aikoii** [reply]
Hey @nivm! One of the options you could consider is creating a Changelog page where you manually list updates. We also allow users to integrate with [Mailchimp](https://www.gitbook.com/integrations/mailchimp). Hopefully one of these options might work for you in the meantime 🙂

**Reply by GeeAVID** [reply]
Our Mailchimp integration does not get emails when you type it and click subscribe. We had configured and authorised in Mailchimp again and again. What would be the issues?

**Comment by ogunreku** [comment]
I just wanted to follow up and ask if there are any updates on whether RSS feed support is being considered or planned.

This would be a huge improvement for both internal teams and external customers, as it allows immediate notifications and aligns with what many competitors already provide. Even knowing if it’s on the roadmap or if there are recommended alternatives would be really helpful.

**Reply by aikoii** [reply]
Hey @ogunreku  👋! Yes, that’s definitely on our list for the future, but I don’t have an ETA to share right now
